AUTHORS.rst

```diff
@@ -45,4 +46,3 @@ Alphabetical list of contributors
 please send an email to the coordinators, or
 `open a pull request for this page <https://github.com/astropy/ccdproc/edit/master/AUTHORS.rst>`_
 in the `ccdproc repository <https://github.com/astropy/ccdproc>`_)
-
```

Comment:
The line shows up in my editor, but doesn't in the git diff. I will add a newline anyway for good measure then.